### PR TITLE
Rewrite exporter

### PR DESCRIPTION
**Pull request by Uxio0**
- Refactor deploy actions
- Remove database dependency
- Remove not needed files
- Increase allocations/snapshot generation time (from half an hour to less than one minute in my laptop)

### CODE REVIEW

**Comment by Uxio0** [comment]
@rmeissner I think it's currently easy to leave `start_date` and duration values `hardcoded`

File: vestings/csv_parser.py [hunk 1]
```diff
@@ -1,86 +1,59 @@
...
-            if start_date is not None:
-                start_date_timestamp = start_date
+            if "startDate" in row.keys():
+                start_date_timestamp = int(parse(row["startDate"]).timestamp())
             else:
-                if "startDate" in row.keys():
-                    start_date_timestamp = parse(row["startDate"]).timestamp()
+                if vesting_type == VestingType.USER_V2:
+                    start_date_timestamp = int(
+                        parse("2022-09-01T10:00:00+00:00").timestamp()
```
Comment (elgatovital) [inline vestings/csv_parser.py:44]: start date could be attribute of the VestingType.

**Review by elgatovital — APPROVED** [review]

File: vestings/csv_parser.py [hunk 2]
```diff
@@ -1,86 +1,59 @@
...
-                duration_weeks = duration
+            start_date_timestamp: int
+
+            if "duration" in row.keys():
+                duration_weeks = int(row["duration"])
             else:
-                if "duration" in row.keys():
-                    duration_weeks = int(row["duration"])
+                if vesting_type in (VestingType.USER_V2, VestingType.INVESTOR):
+                    duration_weeks = 208
```
Comment (elgatovital) [inline vestings/csv_parser.py:35]: default duration value could be attribute of the VestingType

File: vestings/csv_parser.py (hunk 1 above)
Comment (Uxio0) [reply vestings/csv_parser.py:44]: `VestingType` is an `Enum`, but I get what you mean, maybe for next refactor 😉

**Pull request merged**